.github/workflows/Spack.yml

```diff
@@ -60,8 +61,8 @@ jobs:
         spack load g2c
         if [[ "${{ matrix.variants }}" =~ "libs=shared" ]]; then suffix="so" ; else suffix="a"; fi
         ls $G2C_LIB | grep -cE '/libg2c\.'$suffix'$'
-        if [[ "${{ matrix.variants }}" =~ "+png"[+~]?$ && "${{ matrix.variants }}" =~ "libs=shared"[+~]?$ ]]; then ldd $G2C_LIB | grep -c libpng; fi
-        if [[ "${{ matrix.variants }}" =~ "+utils"[+~]?$ ]]; then
+        if [[ "${{ matrix.variants }}" == *"+png"* && "${{ matrix.variants }}" == "libs=shared"* ]]; then ldd $G2C_LIB | grep -c libpng; fi
+        if [[ "${{ matrix.variants }}" == *"+utils"* ]]; then
```

Comment:
@AlexanderRichert-NOAA this was not working and I was confused as heck until I read this: https://stackoverflow.com/questions/229551/how-to-check-if-a-string-contains-a-substring-in-bash

Reply:
Okay good to know. In the future I'll use that as the model for the other libraries.